src/Service.php

```diff
@@ -28,6 +28,8 @@ public function __construct($serviceName)
      */
     public static function getService($serviceName)
     {
+        $serviceName = strtolower($serviceName);
```

Comment:
I went ahead with the hyphenated endpoint name format - so we'll also want to handle converting the camelCase service name to hyphen delimited. I think this would do the trick:

```
$serviceName = strtolower(preg_replace('/([A-Z])/', '-$1', $serviceName))
```